curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-the-filter-method-to-extract-data-from-an-array.english.md

```diff
@@ -7,8 +7,22 @@ forumTopicId: 18179
 
 ## Description
 <section id='description'>
-Another useful array function is <code>Array.prototype.filter()</code>, or simply <code>filter()</code>. The <code>filter</code> method returns a new array which is at most as long as the original array, but usually has fewer items.
-<code>Filter</code> doesn't alter the original array, just like <code>map</code>. It takes a callback function that applies the logic inside the callback on each element of the array. If an element returns true based on the criteria in the callback function, then it is included in the new array.
+Another useful array function is <code>Array.prototype.filter()</code>, or simply <code>filter()</code>. 
+<code>filter</code> calls a function on each element of an array and returns a new array containing only the elements for which that function returns <code>true</code>.  In other words, it filters the array, based on the function passed to it.  Like <code>map</code> it does this without needing to modify the original array.
+The callback function accepts three arguments.  The first argument is the current element being processed.  The second is the index of that element and the third is the array upon which the <code>filter</code> method was called.
+See below for an example using the <code>filter</code> method on the <code>users</code> array to return a new array containing only the users under the age of 30. For simplicity, the example only uses the first argument of the callback.
+
+```js
+const users = [
+  { name: 'John', age: 34 },
+  { name: 'Amy', age: 20 },
+  { name: 'camperCat', age: 10 }
+];
+
+const usersUnder30 = users.filter(user => user.age < 30);
+console.log(usersUnder30); // [ { name: 'Amy', age: 20 }, { name: 'camperCat', age: 10 } ]
+```
+
 </section>
 
 ## Instructions
```

Comment:
Would be a good idea to add to the instructions that the resulting array needs to be saved in the `filteredList` variable.

Reply:
@Manish-Giri I made a new commit that I think improves the instructions for this challenge.  Please let me know what you think.